### DISCUSSION TITLE
Privacy plugin and embedded .html files

### DISCUSSION THREAD

**Discussion by mrsmrynk** [squidfunk/mkdocs-material#7596] [Q&A] [answered]
I use an inline frame (`<iframe>`) to embed .html files in my documentation.
The .html files are generated by [folium](https://python-visualization.github.io/folium/latest/) and download .js files from various CDNs.

Using the privacy plugin unfortunately does not download these external assets.
Is this related to the inline frame and if so, is there a workaround?

Here's an [example .html file](https://github.com/geospaitial-lab/aviary/blob/af3910f1dd374bd08dec9effb517045e8a80e2ef/docs/how_to_guides/api/maps/bounding_box.html#L14)

FYI
The privacy plugin works perfectly fine for any other external asset in my docs.

**Comment by kamilkrzyskow** [comment]
Hello @mrsmrynk,
I believe this is a bug, because the privacy plugin processes HTML data in general for the documentation pages generated from the Markdown file. However, any other HTML files inside the docs are considered static pages, so they're outside of the MkDocs event loop.

That's at least my hunch at the moment, so I suggest to create a full bug report Issue per the guidelines containing an example minimal reproduction. 
- https://squidfunk.github.io/mkdocs-material/contributing/reporting-a-bug/
- https://squidfunk.github.io/mkdocs-material/guides/creating-a-reproduction/

**Reply by mrsmrynk** [reply, marked as answer]
Done in #7600
Thanks for your quick answer 👍🏻
